src/style/row.ts

```diff
@@ -17,6 +19,11 @@ export const row = css`
         background-color: var(--rdg-row-selected-hover-background-color);
       }
     }
+
+    /* Add box-shadow on the last frozen cell */
+    > :nth-last-child(1 of .${cellFrozen}) {
+      box-shadow: var(--rdg-cell-frozen-box-shadow);
+    }
```

Comment:
Does this need to be part of the (Header)Row styles?
We can move it to cell styles, like
```css
&:nth-... {}
```

Reply:
```
&:nth-last-child(1 of &) {
    box-shadow: var(--rdg-cell-frozen-box-shadow);
 }
```
This works